concepticondata/conceptlists/Miller-1984-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Miller-1984-100-1	1	no	not, no	1240	NOT
+Miller-1984-100-2	2	muchos	many	1198	MANY
+Miller-1984-100-3	3	uno	one	1493	ONE
+Miller-1984-100-4	4	dos	two	1498	TWO
+Miller-1984-100-5	5	grande	big	1202	BIG
+Miller-1984-100-6	6	viejo	old	1229	OLD
+Miller-1984-100-7	7	chico	small	1246	SMALL
+Miller-1984-100-8	8	largo	long	1203	LONG
+Miller-1984-100-9	9	nuevo	new	1231	NEW
+Miller-1984-100-10	10	bueno	good	1035	GOOD
+Miller-1984-100-11	11	pesado	heavy	1210	HEAVY
+Miller-1984-100-12	12	seco	dry	1398	DRY
+Miller-1984-100-13	13	frío	cold	2483	COLD (OF WEATHER)
+Miller-1984-100-14	14	caliente	hot	1286	HOT
+Miller-1984-100-15	15	mujer	woman	962	WOMAN
+Miller-1984-100-16	16	hombre	man	1554	MAN
+Miller-1984-100-17	17	gente	person	683	PERSON
+Miller-1984-100-18	18	pez	fish	227	FISH
+Miller-1984-100-19	19	pájaro	bird	937	BIRD
+Miller-1984-100-20	20	perro	dog	2009	DOG
+Miller-1984-100-21	21	piojo	louse	1392	LOUSE
+Miller-1984-100-22	22	culebra, víbora	snake	730	SNAKE
+Miller-1984-100-23	23	árbol, palo	tree	906	TREE
+Miller-1984-100-24	24	corteza, cascara	bark	1204	BARK
+Miller-1984-100-25	25	hoja	leef	628	LEAF
```

Comment:
Is the typo "leef" in the original document?

Reply:
I don't have access to the original data on PharaoHansen-2020-100 (I took the concept list from the corresponding Lexibank repository) but I see @LinguList already checked it. In Miller, however, this typo should be fixed. Thanks for spotting it!